### PR TITLE
removed skip from icx_user

### PR DESCRIPTION
**Pull request by thamaraig**
##### SUMMARY
removed a line which was passing a command i.e skip to Disable page display mode

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
icx_user.py

##### ADDITIONAL INFORMATION
removed a line which was passing a command i.e skip to Disable page display mode

### CODE REVIEW

File: tests/unit/plugins/modules/network/icx/test_icx_user.py [hunk 1]
```diff
@@ -72,10 +72,10 @@ def test_icx_user_update_privilege(self):
     def test_icx_user_update_password(self):
         set_module_args(dict(name='ale1', configured_password='alethea123'))
         if not self.ENV_ICX_USE_DIFF:
-            commands = ['username ale1 password alethea123']  # previous privilage will be added
+            commands = ['username ale1 privilege 5 password alethea123']  # previous privilage will be added
```
Comment (aminvakil) [inline tests/unit/plugins/modules/network/icx/test_icx_user.py:75]:
```suggestion
            commands = ['username ale1 privilege 5 password alethea123']  # previous privilege will be added
```

File: tests/unit/plugins/modules/network/icx/test_icx_user.py [hunk 2]
```diff
@@ -72,10 +72,10 @@ def test_icx_user_update_privilege(self):
...
             self.execute_module(commands=commands, changed=True)
         else:
-            commands = ['username ale1 privilege 5 password alethea123']  # previous privilage will be added
+            commands = ['username ale1 password alethea123']  # previous privilage will be added
```
Comment (aminvakil) [inline tests/unit/plugins/modules/network/icx/test_icx_user.py:78]:
```suggestion
            commands = ['username ale1 password alethea123']  # previous privilege will be added
```

**Review by aminvakil — COMMENTED** [review]
I don't know about this module really and what your PR does really, but would you please fix these typos while you are editing the lines?

**Review by ganeshrn — COMMENTED** [review]
This looks like a breaking change since the default value option is changed. Please add a changelog format eg (https://github.com/ansible-collections/community.network/blob/97b8b2a32640129be962f459a874f172412f833a/changelogs/fragments/199-edgeos-improvements.yaml). The changelogs section can be referred here https://github.com/ansible-collections/community.network/blob/main/changelogs/config.yaml#L12

**Comment by dericcrago** [comment]
closing by request, please let us know if this was in error

**Pull request closed without merging**